substrate/frame/message-queue/src/lib.rs

```diff
@@ -774,6 +776,15 @@ enum MessageExecutionStatus {
 	StackLimitReached,
 }
 
+/// The status of an attempt to process a message.
```

Comment:
The docs seems wrong?

Reply:
Yeah I copied the enum formatting and forgot to update docs. Pushed a new commit to fix.